From 7fe68e6387f592864056530aa8b798f3a54c56b7 Mon Sep 17 00:00:00 2001
From: Drew <drew@localhost>
Date: Sat, 25 Jul 2020 11:25:39 +0200
Subject: [PATCH] fix integer size
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Round bc calculated cursor size to integer, prevent errors on '1.5' and
'2.5' scales.

Signed-off-by: Drew <drew@localhost>
Signed-off-by: Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com>
---
 build.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build.sh b/build.sh
index 9d80f46..3f790b9 100755
--- a/build.sh
+++ b/build.sh
@@ -104,8 +104,8 @@ function generate_in {
 function render {
   name="x$1"
   variant="$2"
-  size=$(echo "$SVG_DIM*$1" | bc)
-  dpi=$(echo "$SVG_DPI*$1" | bc)
+  size=$(echo "$SVG_DIM*$1" | bc | xargs printf %.0f)
+  dpi=$(echo "$SVG_DPI*$1" | bc | xargs printf %.0f)
 
   mkdir -p "$BUILD_DIR/$variant/$name"
   find "$SRC/svg/$variant" -name "*.svg" -type f \
-- 
2.26.2

